balrogscript/src/balrogscript/submitter/cli.py

```diff
@@ -538,3 +538,49 @@ def __init__(self, api_root, auth0_secrets=None):
     def run(self, productName, version, build_number):
         name = get_release_blob_name(productName, version, build_number)
         ReleaseState(name, api_root=self.api_root, auth0_secrets=self.auth0_secrets).set_readonly()
+
+
+class SystemAddonsReleaseCreator(object):
+    def __init__(self, api_root, auth0_secrets):
+        self.auth0_secrets = auth0_secrets
+        self.api_root = api_root
+        self.url = "{}/{}".format(self.api_root, "releases")
+
+    def run(self, manifest):
+        release_blob = {
+            "addons": {},
+            "hashFunction": manifest["hashType"],
+            "name": manifest["releaseName"],
+            "product": "SystemAddons",
+            "schema_version": 5000,
+        }
+        for addon in manifest["addons"]:
+            platforms = {
```

Comment:
We may want to move the platform aliases out into a module constant or config var, since we update them every time we add a new arch, compiler type, or platform. Similar to [`PLATFORMS`](https://github.com/mozilla/morgoth-cli/blob/master/morgoth/xpi.py#L12-L46). We could then deepcopy that into `platforms` before updating it with the `default` dict.